modules/db/src/test/scala/gem/dao/EphemerisDaoSpec.scala

```diff
@@ -183,7 +183,7 @@ class EphemerisDaoSpec extends PropSpec with PropertyChecks with DaoTest {
       val p  = (mʹ.toList.traverse { case (ks, e) =>
         Stream.emits(e.toMap.toList).covary[ConnectionIO]
           .to(EphemerisDao.streamInsert(ks.key, ks.site))
-          .runSync
+          .run
```

Comment:
The `Effect` instance requirement here has been lifted so `runSync` is no longer needed.